extensions/amp-date-picker/0.1/date-picker-common.js

```diff
@@ -33,7 +33,7 @@ export function withDatePickerCommon(WrappedComponent) {
   const moment = requireExternal('moment');
 
   /**
-   * @param {!moment} max
+   * @param {string} max
```

Comment:
can this stay `{!moment}` if you pass `moment(max)` from `isOutsideRange` ?

Reply:
Sure, that could eliminate the extra call to `moment`